src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { Button } from "@mui/material"
+import React from "react"
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+// Check controls from https://storybook.js.org/docs/react/essentials/controls
+
+export default {
```

Comment:
Thank you for making this storybook, so cool!!

Reply:
No problem.